utils/logger/nop.go

```diff
@@ -1,21 +0,0 @@
-package logger
-
-import "net/http"
-
-var _ LoggerI = &NOP{}
-
-type NOP struct{}
```

Comment:
_**Note:**_ there is no longer a need for a no-op logger implementation since, there will always be a non-nil default logger factory available, and logger is immune from constant and concurrent re-initializations.

Reply:
A question, what if we don't want to print any logs? What the non-nil default logger will do?

Reply:
I think a better approach is to keep the NOP{} logger for now, and introduce a particular "mock" logger that would:

a) define the log level 
b) use t.Logf
c) inspect/grep the log for specific output
d) have the option to mute logs

Reply:
To disable logs from the console, one would need to `config.Set("Logger.enableConsole", false)`, but why one would want to disable logs completely?

Reply:
Revived the `logger.NOP` struct, since it appears to be useful for some of us :)